src/server/pps/server/api_server.go

```diff
@@ -2664,7 +2639,7 @@ func (a *apiServer) StopPipeline(ctx context.Context, request *pps.StopPipelineR
 		if _, err := a.env.PfsServer().InspectRepoInTransaction(txnCtx, &pfs.InspectRepoRequest{
```

Comment:
Couldn't the inspect pipeline operation below serve the same purpose as this inspect repo call?

Reply:
Ah, good point